cloudwatch_logger/include/cloudwatch_logger/log_node.h

```diff
@@ -39,8 +39,16 @@ class LogNode : public Service
    * @param min_log_severity the minimum log severity level defined in the configuration file
    *                         logs with severity level equal or above get sent to CloudWatch Logs
    * @param ignore_nodes The set of node names to ignore logs from
+   * @param publish_topic_names whether or not to include topic name information in the log messsages
+   *                            that are uploaded to AWS CloudWatch Logs
    */
-  explicit LogNode(int8_t min_log_severity, std::unordered_set<std::string> ignore_nodes);
+  explicit LogNode(int8_t min_log_severity,
+                   std::unordered_set<std::string> ignore_nodes,
+                   bool publish_topic_names = true);
```

Comment:
Do we need the default value again? 
Are you  concerned that this an API break?

Reply:
Yeah, I added the default value because I am concerned about API break.

I also personally would prefer `publish_topic_names` to be `false` by default, but that would be a behavior change. What do you think?

Reply:
Ideally the options `min_log_severity` and `ignore_nodes` should have been part of an options structure. That we can add more things easily. 

I don't think putting `publish_topic_names` inside `AWS::CloudWatchLogs` makes sense either. 

How about we provide do: 
1. Bump major version 
2. Add a new constructor that takes in an options structure. 
3. People using the old constructor will remain unaffected for the time being. 
4. We can do a planned deprecation of the old constructor later?

Reply:
I think if we're doing a major version bump, then we might as well just remove the old constructor altogether.

Reply:
Does OSRF allow multiple versions of the same ros package? From what I understand the major version bump is symbolic (and for book keeping only) as two versions of the same pacakge cannot exist. By retaining the constructor for a few more releases we can make sure we dont break everyone right away.

Reply:
I don't think their apt repositories keep multiple versions of a ROS package, otherwise we could have rolled back that ROS Melodic core package that had a bug.

If we are keeping the old constructor around, then I think a minor version bump is sufficient, and the major version bump is for when we do finally remove that old constructor.

Reply:
Sure! Either case lets mark the old constructor as [deprecated](https://en.cppreference.com/w/cpp/language/attributes/deprecated) if possible to discourage customers from using it.

Reply:
(The `[[deprecated]]` attribute is new to C++14, and I think our ROS1 packages are supposed to be compatible all the way down to C++11.)